addons/cargo/XEH_postInit.sqf

```diff
@@ -110,7 +125,8 @@ GVAR(initializedVehicleClasses) append _vehicleClassesAddAction;
 
 private _objectClassesAddAction = call (uiNamespace getVariable [QGVAR(initializedItemClasses), {[]}]);
 {
-    [_x, 0, ["ACE_MainActions"], GVAR(objectAction)] call EFUNC(interact_menu,addActionToClass);
+    [_x, 0, ["ACE_MainActions"], GVAR(objectActions) select 0] call EFUNC(interact_menu,addActionToClass);
+    [_x, 0, ["ACE_MainActions"], GVAR(objectActions) select 1] call EFUNC(interact_menu,addActionToClass);
```

Comment:
I think `{...} forEach GVAR(objectActions);` instead for when more actions are added.